notificationExtension/NotificationService.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NotificationService.swift
+//  notificationExtension
+//
+//  Created by NamNH on 28/03/2022.
+//
+
+import UserNotifications
+
+class NotificationService: UNNotificationServiceExtension {
+
+    var contentHandler: ((UNNotificationContent) -> Void)?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/2505580b2b5fa2c2dfc915ebb7d523e53b98aadb/notificationExtension/NotificationService.swift#L12"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>NotificationService.swift:12</code></td>
    </tr>
  </tbody>
</table>